usr/src/common/smbclnt/smbfs_ntacl.c

```diff
@@ -61,21 +64,18 @@
 #define	NT_SD_REVISION	1
 #define	NT_ACL_REVISION	2
 
-#ifdef _KERNEL
+#ifdef __lint
```

Comment:
I don't understand the purpose of this.

Reply:
Ah I see.  Don't we have __unused now? seems like ti would be cleaner than this, which is a really ugly hack.

Reply:
I was getting compiler errors with __unused.
I see now that was probably due to the CSTD Makefile change.
I'll try fixing that and then this can probably go back to using
the inline with an __unused arg.